runtime/testy/studentCode.py

```diff
@@ -9,10 +9,28 @@ def autonomous_main():
   pass
 
 def teleop_setup():
-  pass
+  uids = Robot._getAllUIDs()
```

Comment:
We could just have two files: `student_code.py` as a barebones template, and `test_code.py` for our drive test code.